inc/customizer.php

```diff
@@ -265,11 +244,11 @@ public function render_content() {
 			'type'        => 'radio',
 			'label'       => __( 'Logo style', 'oblique' ),
 			'description' => __( 'This option applies only if you are using a logo', 'oblique' ),
-			'section'     => 'oblique_header',
+			'section'     => 'title_tagline',
 			'priority'    => 13,
 			'choices'     => array(
 				'hide-title' => __( 'Only logo', 'oblique' ),
-				'show-title' => __( 'Logo plus site title&amp;description', 'oblique' ),
+				'show-title' => __( 'Logo plus site title &amp; description', 'oblique' ),
```

Comment:
here we need to keep the same string ( even though is not correct ), because this is a free theme on wporg and we want to have the percentage of translated strings as high as possible and changing strings is not helping that https://translate.wordpress.org/projects/wp-themes/oblique